### ISSUE TITLE
Incorrect link used for Common Issue section

### ISSUE THREAD

**Issue by HartD92** [MicrosoftDocs/memdocs#4164]
[Enter feedback here]

In section for common issues, the link for "Unable to discover printers on the client" routes to a private visualstudio.com instance that the public does not have access to. Please migrate to a Microsoft Docs page.

"For more information and possible steps, go to [Unable to discover printers on the client](https://supportability.visualstudio.com/WindowsUserExperience/_wiki/wikis/WindowsUserExperience/725155/Unable-to-discover-printers-on-the-client)."

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 2c561ca3-51c0-3256-4826-9c1683110c69
* Version Independent ID: 2c561ca3-51c0-3256-4826-9c1683110c69
* Content: [Configure Universal Print policy using settings catalog in Microsoft Intune](https://learn.microsoft.com/en-us/mem/intune/configuration/settings-catalog-printer-provisioning)
* Content Source: [memdocs/intune/configuration/settings-catalog-printer-provisioning.md](https://github.com/MicrosoftDocs/memdocs/blob/main/memdocs/intune/configuration/settings-catalog-printer-provisioning.md)
* Service: **microsoft-intune**
* Sub-service: **configuration**
* Technology: ****
* GitHub Login: @MandiOhlinger
* Microsoft Alias: **mandia**

**Comment by MandiOhlinger** [comment]
@HartD92 Good catch. Thank you! The link is removed. The changes should be live later today.
